src/Symfony/Component/DependencyInjection/Attribute/TaggedItem.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+/**
+ * An attribute to tell under which index a class should be found in tagged iterators/locators.
```

Comment:
```suggestion
 * An attribute to declare the index and priority of a class when used in tagged iterators/locators.
```

(just a suggestion, to also get "priority" in there)

Reply:
The priority is mentioned, so I don't get what you mean sorry.

Reply:
The phpdoc description doesn't (or I'm misreading it). In any case, Ryan's suggestion is fine as well

Reply:
Oh, I miss read this was a suggestion, got it, thanks.